### PR TITLE
Mccalluc/refactor panel list

### PR DESCRIPTION
**Pull request by mccalluc**
I started off trying to reuse my-lists components, but they didn't seem to actually be appropriate here.

- I could be wrong, but I don't think we want a fixed height div here: It makes sense if there are multiple scrolling divs on the page, but here, since it's the only one, it just makes scrolling trickier.
- The panel there is really specific to the my-lists content, and there isn't much left if it is generalized enough to work for this case.

I have not touched the content in the list -- I want to approach that separately, since there need to be more changes to the structure that I want Juan's buy-in on.
Screen Shot 2022-08-03 at 3 02 11 PM

### CODE REVIEW

File: context/app/static/js/components/workspaces/style.js [hunk 1]
```diff
@@ -8,4 +8,11 @@ const StyledButton = styled(WhiteBackgroundIconButton)`
...
 `;
 
-export { StyledButton };
+const Left = styled.div`
+  flex-shrink: 0;
+`;
+const Right = styled.div`
+  flex-shrink: 0;
+`;
+
```
Comment (john-conroy) [inline context/app/static/js/components/workspaces/style.js:17]: If the names are useful, I'd create a single styled component and then create `Left` and `Right` from that.

File: context/app/static/js/components/workspaces/style.js [hunk 2]
```diff
@@ -8,4 +8,11 @@ const StyledButton = styled(WhiteBackgroundIconButton)`
   height: 36px;
 `;
 
-export { StyledButton };
+const Left = styled.div`
+  flex-shrink: 0;
```
Comment (john-conroy) [inline context/app/static/js/components/workspaces/style.js:12]: What does this look like without `flex-shrink: 0`? Struggling to visualize the impact.

**Review by john-conroy — COMMENTED** [review]
I think in previous workspace reviews, I've been considering the PRs as prototyping and have been less scrutinizing. Are we now moving out of that phase?

File: context/app/static/js/components/workspaces/style.js (hunk 2 above)
Comment (mccalluc) [reply context/app/static/js/components/workspaces/style.js]: Actually, plain divs work; Replaced.

**Comment by mccalluc** [comment]
> Are we now moving out of that phase?

I'd say moving out, but not out yet! I've filed a follow-up (#2803) that changes the information we show, and I want to get Juan's buy-in on that before going too deep on the UI polishing.

**Pull request merged**